### ISSUE TITLE
[Feature Request] Add option to perform explicit import without settings merge (i.e. clear app settings before import)

### ISSUE THREAD

**Issue by jesselai** [M66B/XPrivacy#1028]
It would be nice to have the option when performing an import to explicitly use the settings that are imported per app rather than merging the current settings with the imported settings.  This is useful in situations when sharing settings across devices and a global settings clear isn't desired.  The option could either be chosen at import time or done via a checkbox in settings. See a use case description below:

I have several devices, phones and tablets, that have different apps installed on them. There are many apps that are the same, so I've been tweaking the settings on my phone, then exporting and going to import on my tablet for example. The export/import works fine, but I noticed one issue with it, probably by design, but it does make it difficult for my workflow. When doing an import, it does the conservative approach and keeps items blocked, even if they are not blocked on the new import. Would it be possible to create a setting used in the import that would allow a clear before import for the imported apps? That way the imported settings would match.

I don't want to do a global clear, because there are other apps that are not included in the import that I want to keep the settings for. What I currently do is go through and manually clear the apps that will be imported, but it would be a nice feature to have that in the importer.
